.ruby-version

```diff
@@ -0,0 +1 @@
+ruby-2.2.2
```

Comment:
Early i deployed applications to locum (russian hosting). Locum used latest version is 2.2.2. On the future i will use latest version ruby.